### ISSUE TITLE
How to enable libfuse debug

### ISSUE THREAD

**Issue by RRRadicalEdward** [cberner/fuser#251]
Hi! I don't understand how I can enable libfuse debug. I tried pass `MountOption` as `MountOption::CUSTOM("debug".to_string())` and `MountOption::CUSTOM("-f".to_string())`, but they seem to do nothing.
Any help appreciated  🙂

**Comment by cberner** [comment]
fuser only uses a small set of libfuse API calls (these for [libfuse2](https://github.com/cberner/fuser/blob/56ecbb069dda0b9a5789559e6e2795fe08b9e34a/src/mnt/fuse2_sys.rs#L24-L33) and these for [libfuse3](https://github.com/cberner/fuser/blob/56ecbb069dda0b9a5789559e6e2795fe08b9e34a/src/mnt/fuse3_sys.rs#L15-L30)), so they might not have any debug logging

**Comment by RRRadicalEdward** [comment]
Fuse logging can be enabled if path `MountOption::CUSTOM("--debug")` and modify a little bit `fuser` to pass `CUSTOM` options without "-o" to `fuse`.
![image](https://github.com/cberner/fuser/assets/55661041/aceafed1-803e-4fc1-b844-84d4ddfd8e47)
But `fuse` low-level API does little of logging itself. It's not really useful for debugging. 

I'm implementing virtual filesystem using fuser. And actully my issue that I can't do `cd` in my filisystem after mounting. 
`fuser` and `fuse` doesn't log anything useful to fix the issue. After digging more in `fuser` and `fuse`, I noticed that a operation (e.g `readdir`) result(pid, guid, ino, etc) is transmitted trough a channel. I found that `fuse` sends a message with error code  though this channel if an error happens, but I didn't find how `fuser` handles such messages. Does it? Can it be this types of message aren't simply parsed by `fuser`
